learn/app-development/widgets/datalive/live-form/liveform-layouts.md

```diff
@@ -11,6 +11,7 @@ Live Form **layouts** include:
 1. Column Form with left-aligned Captions
 2. Column Form with right-aligned Captions
 3. Column Form with top-aligned Captions
+4. Column Form with floating Captions
```

Comment:
Lets add in brackets (Angular material style)

Reply:
Added